src/info.jl

```diff
@@ -51,3 +33,49 @@ By default, returns `nothing` as info.
 function update_info(up::Updater, b, a, o)
     return update(up, b, a, o), nothing
 end
+
+"""
+    add_infonode(ddn::DDNStructure)
+
+Create a new DDNStructure object with a new node labeled :info with parents :s and :a
+"""
+function add_infonode(ddn) # for DDNStructure, but it is not declared in v0.7.3, so there is not annotation
+    add_node(ddn, :info, ConstantDDNNode(nothing), (:s, :a))
+end
+
+function add_infonode(ddn::POMDPs.DDNStructureV7{nodenames}) where nodenames
+    return POMDPs.DDNStructureV7{(nodenames..., :info)}()
+end
+
+###############################################################
+# Note all generate functions will be deprecated in POMDPs v0.8
+###############################################################
+
+
+if DDNStructure(MDP) isa POMDPs.DDNStructureV7
```

Comment:
shouldn't we check on the POMDPs.jl version instead?

Reply:
That's what I initially though, but it didn't seem like there was a good way to do that, so this seems like a reliable enough proxy

Reply:
`if Pkg.installed()["POMDPs"] < v"0.8.0"`
something like that?

Reply:
I don't know how to reply to your comment on this below.

The problem with Pkg.installed() is that it can take a really long time (or at least it could in the past)

Reply:
and it adds Pkg as a dependency (which is probably fine)